jekyll/explorer.md

```diff
@@ -12,11 +12,13 @@ Climate Explorer is currently in beta and will be generally available in mid 202
 If you do not yet have a Climate Explorer account, please contact [sales@sustglobal.com](mailto:sales@sustglobal.com).
 
 
-# Create New Portfolios
+# Projects in Climate Explorer
+
+As a user of Climate Explorer, you can manage collections of physical assets as "portfolios", each of which can be represented as a CSV file. A project is a logical grouping of portfolios with either viewer (read only), editor (ability to create new portfolios) or owner (ability to create new users with access to the portfolio). As a user, you will have access to multiple projects. For example, we will provide you with viewer access to a demo project containing demo portfolios. We will also provide you with editor access to new empty projects with editor access where you can create your own portfolios.
```

Comment:
Let's remove the reference to 'owner' for now since that role does not actually exist in the platform yet.